doc/source/reference/commands/select.rst

```diff
@@ -2453,6 +2455,55 @@ specify one key as ``drilldowns[${LABEL}].keys`` like ``--drilldowns[tag].keys
 tag``. You should use ``_key`` for the case. It's the same rule in
 :ref:`select-drilldown-output-columns`.
 
+.. _select-drilldowns-label-table:
+
+``drilldowns[${LABEL}].table``
+""""""""""""""""""""""""""""""
+
+.. versionadded:: 6.0.2
+
+Specifies ``${LABLE}`` of other ``drilldown``, ``drilldowns`` or ``slices``.
+
+The result of the specified ``${LABLE}`` is drilldowned,
+which means that this paramter enables nested aggregate calculations and groups in drilldown.
+
+Here is an example to execute nested drilldown by tag and then by category.
+
+.. groonga-command
+.. include:: ../../example/reference/commands/select/drilldowns_label_table.log
+.. table_create Tags TABLE_PAT_KEY ShortText
+.. column_create Tags category COLUMN_SCALAR ShortText
+.. table_create Memos TABLE_HASH_KEY ShortText
+.. column_create Memos tag COLUMN_SCALAR Tags
+.. load --table Memos
+.. [
+.. {"_key": "Groonga is fast!", "tag": "Groonga"},
+.. {"_key": "Groonga sticker!", "tag": "Groonga"},
+.. {"_key": "Mroonga sticker!", "tag": "Mroonga"},
+.. {"_key": "Rroonga is fast!", "tag": "Rroonga"}
+.. ]
+.. load --table Tags
+.. [
+.. {"_key": "Groonga", "category": "C/C++"},
+.. {"_key": "Mroonga", "category": "C/C++"},
+.. {"_key": "PGroonga", "category": "C/C++"},
+.. {"_key": "Rroonga", "category": "Ruby"}
+.. ]
+.. select Memos \
+..   --drilldowns[label_tag].keys tag \
```

Comment:
Can we use better name for `label_tag`?
I don't want users to use `label_XXX` for label.

Reply:
I have changed to simply use Tag and Category for these `drilldowns` table.